redis/cluster.py

```diff
@@ -968,6 +968,14 @@ def determine_slot(self, *args):
             if len(eval_keys) == 0:
                 return random.randrange(0, REDIS_CLUSTER_HASH_SLOTS)
             keys = eval_keys
+        elif command == "ZUNIONSTORE":
```

Comment:
```
georadius
georadiusbymember
migrate
sort
sort_ro
zdiffstore
zinterstore
```

shouldn't we raise an exception for above commands?

Reply:
I assume that we are not using those commands, but just in case.

Reply:
yeap, i'm trying to block those commands in soda code, let me write the PR link once it's ready

Reply:
got it, please be careful that we should block direct call like `execute_command(sort)`

Reply:
https://github.com/sendbird/soda/pull/16246